enterprise/dev/ci/ci/pipeline-steps.go

```diff
@@ -97,22 +97,22 @@ func addBrowserExt(pipeline *bk.Pipeline) {
 	// Browser extension tests
 	pipeline.AddStep(":jest::chrome:",
 		bk.Cmd("dev/ci/yarn-test.sh browser"),
-		bk.ArtifactPaths("browser/coverage/coverage-final.json"))
+		bk.Cmd("bash <(curl -s https://codecov.io/bash) -c -F browser -F unit"))
 }
 
 // Tests the precise code intel system.
 func addPreciseCodeIntelSystem(pipeline *bk.Pipeline) {
 	pipeline.AddStep(":jest:",
 		bk.Cmd("dev/ci/yarn-test-separate.sh cmd/precise-code-intel"),
-		bk.ArtifactPaths("cmd/precise-code-intel/coverage/coverage-final.json"))
+		bk.Cmd("bash <(curl -s https://codecov.io/bash) -c -F codeintel -F unit"))
 }
 
 // Adds the shared frontend tests (shared between the web app and browser extension).
 func addSharedTests(pipeline *bk.Pipeline) {
 	// Shared tests
 	pipeline.AddStep(":jest:",
 		bk.Cmd("dev/ci/yarn-test.sh shared"),
-		bk.ArtifactPaths("shared/coverage/coverage-final.json"))
+		bk.Cmd("bash <(curl -s https://codecov.io/bash) -c -F sharedtypescript -F unit"))
```

Comment:
I'd like to keep these short
```suggestion
		bk.Cmd("bash <(curl -s https://codecov.io/bash) -c -F sharedts -F unit"))
```